cli/tests/test_operations.rs

```diff
@@ -203,6 +203,106 @@ fn test_op_log_no_graph() {
     "#);
 }
 
+#[test]
+fn test_op_log_reversed() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["git", "init", "repo"]);
+    let repo_path = test_env.env_root().join("repo");
+    test_env.jj_cmd_ok(&repo_path, &["describe", "-m", "description 0"]);
+
+    let stdout = test_env.jj_cmd_success(
+        &repo_path,
+        &[
+            "op",
+            "log",
+            "--config=template-aliases.'format_time_range(x)'=x",
```

Comment:
nit: I think this alias isn't needed anymore. Relative timestamp is disabled globally in tests.